articles/sendgrid-dotnet-how-to-send-email.md

```diff
@@ -117,7 +117,7 @@ You may store these credentials via your Azure Portal by clicking Application se
     var apiKey = System.Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
     var client = new SendGridClient(apiKey);
 
-The following examples show how to send a message using the Web API.
+The following examples show how to send a message using a console application.
```

Comment:
How about:

The following examples show how to send an email message using the SendGrid Web API with a console application.

Reply:
ok i will modify it . Thanks for the input :)